app-editors/apostrophe/spec

```diff
@@ -0,0 +1,6 @@
+VER=3.2
+SRCS="git::commit=tags/v$VER;rename=apostrophe::https://gitlab.gnome.org/World/apostrophe.git \
+      git::commit=09505903003285573dc12d971a5e4c621fe14cae;rename=reveal.js::https://github.com/hakimel/reveal.js.git"
```

Comment:
Where does this commit hash come from? If it's just the HEAD commit then leave a comment to say so (as where to find the commit hash). It is also recommended to move the hardcoded hash to its own variable and reference that variable in `SRCS`.